extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -491,7 +492,7 @@ export class AmpSlideScroll extends BaseSlides {
       toScrollLeft = 0;
     }
     return this.animateScrollLeft_(currentScrollLeft, toScrollLeft).then(() => {
-      this.updateOnScroll_(toScrollLeft);
+      this.updateOnScroll_(toScrollLeft, opt_trust || ActionTrust.LOW);
```

Comment:
Just make `updateOnScroll_` default to `LOW`.

Also use newer optional arg syntax like `(arg, arg, opt_trust = ActionTrust.LOW)`.